curvefs/src/client/kvclient/kvclient_manager.cpp

```diff
@@ -52,29 +61,51 @@ void KVClientManager::Uninit() {
 
 void KVClientManager::Set(std::shared_ptr<SetKVCacheTask> task) {
     threadPool_.Enqueue([task, this]() {
-        LatencyGuard guard(&kvClientMetric_.kvClientSet.latency);
+        LatencyGuard guard(&kvClientManagerMetric_->set.latency);
 
         std::string error_log;
         task->res =
             client_->Set(task->key, task->value, task->length, &error_log);
-        task->timer.stop();
-        ONRETURN(Set, task->res);
-
-        task->done(task);
+        if (task->res) {
+            kvClientManagerMetric_->count << 1;
```

Comment:
Now that there are hits and misses, is count redundant?

Reply:
> Now that there are hits and misses, is count redundant?

count means `set`

hits and miss are `get`